ke/compiler.py

```diff
@@ -89,6 +89,47 @@
     builtin_macros["#" + short] = builtin_macros["#" + long]
 
 
+def compile_separate(separate, expr):
```

Comment:
please install the pre-commit hooks (`pip install pre-commit && pre-commit install`) and fix all issues once (`SKIP=cargo-check,fmt pre-commit run`)